### PR TITLE
Adds isbn-verifier exercise

### PR DESCRIPTION
**Pull request by amscotti**
Hi, this PR is to add a new exercise for `isbn-verifier`, [here is a link to the problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises/isbn-verifier)

I'm still learning Dart, so feel free to point out anything that needs to be change or improved. I'm also happy to change the `config.json` to better fit your layout for the track.

Thanks! :)

### CODE REVIEW

File: exercises/isbn-verifier/lib/example.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class IsbnVerifier {
```
Comment (Stargator) [inline exercises/isbn-verifier/lib/example.dart]: I would suggest removing the class and just allowing the method to be accessible.

File: exercises/isbn-verifier/lib/example.dart [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  final RegExp formatChecker = new RegExp(r'^\d-?\d{3}-?\d{5}-?[\dX]$');
+  final RegExp cleaner = new RegExp(r'\W');
```
Comment (Stargator) [inline exercises/isbn-verifier/lib/example.dart]: If the above is taken, then these variable should be private.

File: config.json [hunk 3]
```diff
@@ -313,6 +313,18 @@
         "strings",
         "type_conversion"
       ]
+    },
+    {
+      "slug": "isbn-verifier",
+      "uuid": "c52dfe2b-5634-4c0c-8ea2-3ca86e5d9a33",
+      "core": false,
```
Comment (Stargator) [inline config.json]: I think this would be a good core exercise. Otherwise, I think this PR is ready.

Comment (amscotti) [reply config.json]: @Stargator Sounds good, where did you want to have this exercise take place in this list of core exercises? Add to the end?

Comment (Stargator) [reply config.json]: It can stay where it is.

**Comment by amscotti** [comment]
@Stargator Updated.

**Pull request merged**